lib/addons/p5.dom.js

```diff
@@ -58,12 +58,23 @@
    * </code></div>
    *
    */
-  p5.prototype.select = function (e) {
+  p5.prototype.select = function (e, p) {
     var res;
     var str;
+    var container = document;
+    if (typeof p === 'string'){
+      if(p[0] !== '#'){
+        container = document.getElementById(p);
+      }else {
+        p = p.slice(1);
+        container = document.getElementById(p);
+      }
+    }else if (typeof p === 'object'){
+      container = document.getElementById(p.elt.id);
+    }
```

Comment:
This is actually not necessary, since we already have access to the element, so we can simplify line 73 to be:
```javascript
container = p.elt;
```